File: csharp/ex007.cs
// Crie um algoritmo que leia um número real e mostre na tela o seu dobro e a
// sua terça parte.
// Ex:
// Digite um número: 3.5
// O dobro de 3.5 é 7.0
// A terça parte de 3.5 é 1.16666

class Ex007
{
  public double numero;
  
  public void ShowThirdAndDouble()
  {
    Console.WriteLine("Digite um numero: ");
    numero = Convert.ToDouble(Console.ReadLine());

    Console.Clear();
    Console.WriteLine($"O dobro de {numero} é {numero * 2}");
    Console.WriteLine($"A terça parte de {numero} é {Math.Round(numero/3,4)}");
  }
}

File: typescript/src/ex007.ts
// Crie um algoritmo que leia um número real e mostre na tela o seu dobro e a
// sua terça parte.
// Ex:
// Digite um número: 3.5
// O dobro de 3.5 é 7.0
// A terça parte de 3.5 é 1.16666

var readline = require('readline-sync')

let numReal : number = Number(readline.question("Digite um numero: "));
let dobro : number = numReal *2;
let tercParte: number = numReal / 3;

console.log(`O dobro de ${numReal} é ${dobro}`);
console.log(`A terça parte de ${numReal} é ${tercParte}`);